src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const { personSlug = '' } = useParams();
+  const havePeople = people.length > 0;
+
+  const loadPeople = useCallback(
+    async () => {
+      setIsLoading(true);
+      try {
+        const loadedPeople = await getPeople();
+
+        setPeople(loadedPeople);
+      } catch {
+        setHasError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    }, [getPeople],
+  );
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+
+          {hasError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {!havePeople && !isLoading && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
+
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <PeopleTable
+              people={people}
+              selectedPerson={personSlug}
+            />
+          )}
```

Comment:
Show table only if data is loaded and people array is not empty

Reply:
![Screenshot 2023-04-21 at 16 47 25](https://user-images.githubusercontent.com/84275496/233653833-198123b0-0981-44d1-a47c-d5e151a153f1.png)